res/images/library/ic_library_relations.svg

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg id="svg2" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 32 32">
+  <defs>
+    <style>
+      .cls-1 {
+        stroke: url(#linear-gradient);
+      }
+
+      .cls-1, .cls-2, .cls-3 {
+        fill: none;
+        stroke-linecap: round;
+        stroke-miterlimit: 10;
+        stroke-width: 3px;
+      }
+
+      .cls-2 {
+        stroke: url(#linear-gradient-2);
+      }
+
+      .cls-3 {
+        stroke: url(#linear-gradient-3);
+      }
+    </style>
+    <linearGradient id="linear-gradient" x1="-62.15" y1="-32.24" x2="-46.47" y2="-32.24" gradientTransform="translate(26.51 82.3) rotate(45)" gradientUnits="userSpaceOnUse">
+      <stop offset="0" stop-color="#f60"/>
+      <stop offset="1" stop-color="#de5800"/>
+    </linearGradient>
+    <linearGradient id="linear-gradient-2" x1="-62.15" y1="-46.66" x2="-46.47" y2="-46.66" xlink:href="#linear-gradient"/>
+    <linearGradient id="linear-gradient-3" x1="-55.81" y1="-39.64" x2="-52.81" y2="-39.64" xlink:href="#linear-gradient"/>
+  </defs>
+  <path class="cls-1" d="M9.97,13.07l-2.9,2.9c-2.31,2.31-2.31,6.07,0,8.38l.58.58c2.31,2.31,6.07,2.31,8.38,0l2.9-2.9"/>
+  <path class="cls-2" d="M22.03,18.93l2.9-2.9c2.31-2.31,2.31-6.07,0-8.38l-.58-.58c-2.31-2.31-6.07-2.31-8.38,0l-2.9,2.9"/>
+  <line class="cls-3" x1="19.03" y1="12.97" x2="13.25" y2="18.75"/>
+</svg>
```

Comment:
I think this shall be a link icon, right? I am probably to old that this works for me. A bit too abstract.
Can we adjust it towards this: 
https://www.istockphoto.com/de/vektor/link-symbol-gm1064005640-284472456

I think just a bit rounding into the inner chain ring helps to get the desired imagination